src/btouch.cs

```diff
@@ -520,11 +498,10 @@ int Main3 (string [] args)
 			AttributeManager = new AttributeManager (this);
 			Frameworks = new Frameworks (CurrentPlatform);
 
-			Assembly corlib_assembly = universe.LoadFile (LocateAssembly ("mscorlib"));
 			// Explicitly load our attribute library so that IKVM doesn't try (and fail) to find it.
-			universe.LoadFile (GetAttributeLibraryPath ());
+			universe.LoadFromAssemblyPath (GetAttributeLibraryPath ());
```

Comment:
Do you need this line at all, given that you do this further above:

```csharp
assemblyPaths.Add (GetAttributeLibraryPath ());
```

Reply:
Likely not. I tried to keep the code changes to minimum.

Reply:
Removing the explicit load leads to `error BI1055: bgen: Internal error: failed to convert type 'BaseTypeAttribute, bgen, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.`. I'll investigate it but I reverted the removal for now.